config/config.go

```diff
@@ -71,55 +70,48 @@ type GlobalFlagValues struct {
 	NoColor          bool
 }
 
-func DefaultConfig() *Config {
+func DefaultConfig() Config {
```

Comment:
Why did we convert return type to object instead of pointer?

Reply:
In most cases it's better to return a non-pointer value. In this specific case, returning the address of `Config` causes it to escape to heap, but returning it doesn't (we don't care about performance in many code paths in CLC, so that's not very important).